schema/deploy/shipping/views.sql

```diff
@@ -3212,27 +3235,85 @@ create or replace view shipping.__uw_priority_queue_v1 as (
-                when tier in ('2', '3') then 8
-                else null
-              end as priority,
-            case
-                when coalesce(latest_invite_date, latest_collection_date) is null and tier = '1' then 'tier_1_baseline'
-                when coalesce(latest_invite_date, latest_collection_date) is null and tier in ('2', '3') then 'tier_2_and_3_baseline'
-                when tier = '1' then 'tier_1_surveillance'
-                when tier in ('2', '3') then 'tier_2_and_3_surveillance'
-                else null
-            end as priority_reason
+            2 as priority,
+            'uw_housing_resident' as priority_reason
         from uw_enrollments
-        -- Filter to enrollments that have never been invited to test or last invite was over 3 days before today
-        where (latest_invite_date is null or latest_invite_date < current_date - interval '3 days')
-        -- Filter to enrollments have never had a sample collected or last sample collection was over 3 days before today
-        and (latest_collection_date is null or latest_collection_date < current_date - interval '3 days')
+        -- Filter to the housing group for today
+        -- CHR 97 is 'a'
+        -- DOW is 0 through 6, starting on Sunday; therefore, Sunday is group 'a'
+        where uw_housing_group = CHR(EXTRACT(DOW from current_date)::int + 97)
+        -- Filter to encounters for participants whose last invite was before today
+        and (latest_invite_date is null or latest_invite_date < current_date)
+        -- Filter for participants who have not tested positive with us in the past 90 days
+        and (latest_positive_hcov19_collection_date is null or latest_positive_hcov19_collection_date < current_date - interval '90 days')
+        -- Filter for participants who have not tested positive somewhere else in the past 90 days
+        and (latest_prior_test_positive_date is null or latest_prior_test_positive_date < current_date - interval '90 days')
+        -- Filter for participants who will receive alerts from REDCap
+        and (alerts_off is null or alerts_off <> 'off')
+	),
+
+    -- Select enrollments for baseline testing
+    baseline as (
+        select
+            redcap_url,
+            redcap_project_id,
+            redcap_record_id,
+            redcap_event_name,
+            redcap_repeat_instance,
+            encountered,
+            individual,
+            tier,
+            latest_invite_date,
+            latest_collection_date,
+            4 as priority,
+            'baseline' as priority_reason
+        from uw_enrollments
+        -- Filter to participants who enrolled after this date when
+        -- baseline test invitations were handled manually
+        where encountered > '2020-12-15'
+         -- Filter to enrollments have never had a sample collected
+        and latest_collection_date is null
+        -- Filter to enrollments that have been invited up to 2 times
+        and times_invited < 2
```

Comment:
This could be renamed to invitation_count for clarity (in code, I often see time meaning time of day)

Reply:
I renamed that.